template-helpers/.eslintrc

```diff
@@ -0,0 +1,5 @@
+{
+  "rules": {
+    "camelcase": ["error", {properties: "never"}]
+  }
+}
```

Comment:
Same as comment way above, should this be in the global .eslint config as an override (or disabled globally since it seems to be a very common pattern to ignore this anyways).